linter/whitelists/de.txt

```diff
@@ -508,4 +508,5 @@ Dateibaum
 Bedingungsfeldern
 Bedingungsfelder
 Catalog
-Manager
\ No newline at end of file
+Manager
+Slack
```

Comment:
This could go to default.tx instead because it‘s a product name and not language specific :) While you‘re on it, do you mind checking for other stuff too? „Manager“ for example could certainly go to default.txt as well :)

Reply:
Hey ho, totally right, but I just ran a diff and de and en only differ in like five terms. There is plenty of german in the english file as well. So I wonder where to start…

And what about such generic terms as Asset, Attachement, Bundle, ... I feel that they should go to default.txt as well, if Manager goes there?

Reply:
Well the way I think about it is that I want names and general terms in the `default` file.
`Manager` is such an example, because it likely refers to `Contao Manager`. Or `Bundle` because it's a term that will be used in German, French, Italian etc. as it's a technical term.
However, `asset` is not something that belongs to `default`, really. There are proper translations for it in German, French etc.
Same applies for `Attachment`.

It's hard to find the ideal rule but that's how I think it **should** be 😄

Reply:
Agreed. I started moving some terms around but I sense this might cause some discussion I only moved Slack to default.txt for now so we may close this PR :-) Will open another than to clean up the filter list.

Reply:
Perfect!!